spec/factories/openscap_rule_result.rb

```diff
@@ -0,0 +1,3 @@
+FactoryGirl.define do
+  factory :openscap_rule_result_for_test, :class => OpenscapRuleResult
```

Comment:
👎  I'd prefer to have this PR add the base class factories to this repo (where they belong) with the correct name and a PR to remove it from the providers repo.  If the two PR's are linked together, we can coordinate the merge so that there are no test failures.

Reply:
Ok, I will make this happen